traffic_ops/testing/api/v4/cachegroups_parameters_test.go

```diff
@@ -79,6 +90,129 @@ func CreateTestCacheGroupParameters(t *testing.T) {
 	testData.CacheGroupParameterRequests = append(testData.CacheGroupParameterRequests, resp.Response...)
 }
 
+func CreateTestCacheGroupParametersAlreadyExist(t *testing.T) {
+
+	resp, _, _ := TOSession.GetAllCacheGroupParameters(client.RequestOptions{})
+	cachegroupParameters := resp.Response
+	getAllcgparameters := cachegroupParameters.CacheGroupParameters
+	if len(getAllcgparameters) < 1 {
+		t.Fatalf("No existing cachegroup parameters available to test duplicate functionality")
+	}
+	if getAllcgparameters[0].Parameter == nil && *getAllcgparameters[0].Parameter != 0 {
```

Comment:
The second part of this check will always segfault if it gets checked. Also, I think a Parameter ID of 0 is always invalid, so I don't think we want to enforce that. Seems like the second part of this check just shouldn't be there.

Reply:
removed the second part of the validation.